packages/react-components/react-search/stories/SearchBox/SearchBoxDefault.stories.tsx

```diff
@@ -3,4 +3,17 @@ import { SearchBox, SearchBoxProps } from '@fluentui/react-search';
 
 import { FilterRegular } from '@fluentui/react-icons';
 
-export const Default = (props: Partial<SearchBoxProps>) => <SearchBox {...props} contentAfter={<FilterRegular />} />;
+export const Default = (props: Partial<SearchBoxProps>) => (
+  <>
+    <SearchBox {...props} contentAfter={<FilterRegular />} size="small" placeholder="small" />
+    <SearchBox {...props} contentAfter={<FilterRegular />} size="medium" placeholder="medium" />
+    <SearchBox {...props} contentAfter={<FilterRegular />} size="large" placeholder="large" />
+    <SearchBox {...props} contentAfter={null} placeholder="no contentAfter" />
+    <SearchBox {...props} contentAfter={<FilterRegular />} disabled placeholder="disabled" />
+    <SearchBox
+      {...props}
+      contentAfter={<FilterRegular tabIndex={0} onClick={() => console.log('clicked')} />}
+      placeholder="contentAfter button"
+    />
+  </>
```

Comment:
For the future, I think you'll want to split these out to different stories. That can be done in another PR though 👍